app/equipment/armour.py

```diff
@@ -0,0 +1,11 @@
+from app.keys import KeysArmour
+
+
+class Armour:
+    def __init__(self, list_armour: list) -> None:
+        protect = []

```

Comment:
why create a list? 
if you just want to calculate a sum use an integer variable